drivers/adodb-db2.inc.php

```diff
@@ -133,7 +133,9 @@ private function doDB2Connect($argDSN, $argUsername, $argPassword, $argDatabasen
 		if ($this->debug){
 			if (strcmp($argDSN,'*LOCAL') == 0)
 			{
-				$connectMessage = '*LOCAL connection'; 
+				$connectMessage = '*LOCAL connection';
+				$schema = $argDatabasename;
+				$argDatabasename = '';
```

Comment:
The schema assignment is wrapped in `if ($this->debug)`, don't you want to do this also when not in debug mode ?